cmake/KokkosConfig.cmake.in

```diff
@@ -62,3 +62,7 @@ ELSEIF(@Kokkos_ENABLE_CUDA@ AND NOT "separable_compilation" IN_LIST Kokkos_FIND_
 ENDIF()
 
 set(Kokkos_COMPILE_LANGUAGE @KOKKOS_COMPILE_LANGUAGE@)
+
+# Kokkos_<INFO> vars
```

Comment:
What are `Kokkos_<INFO>` variables?

I figured it out after I checkout your branch and inspected the file that was configured but nevertheless I think that comment is somewhat cryptic.

Reply:
Kokkos can change this comment to anything it likes.  When users see what this is, this is not going to be a surprise.